### PR TITLE
The package needs some fixes to work with more recent rq versions.

### PR DESCRIPTION
**Pull request by konstantint**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,13 +18,13 @@ Installation and Use
 You can install this package directly from GitHub with pip:
 
 ```bash
-$ pip -e git+https://github.com/michaelbrooks/rq-win.git#egg=rq-win
+$ pip install git+https://github.com/michaelbrooks/rq-win.git#egg=rq-win
```
Comment (michaelbrooks) [inline README.md]: Does this work now? I thought you had to use -e to point at a git repo.

**Comment by michaelbrooks** [comment]
Looks good. I'll merge it.

**Comment by michaelbrooks** [comment]
Added a release tag for 0.3.13

File: README.md (hunk 1 above)
Comment (konstantint) [inline README.md]: Things work this way on my machine (but it's probably trivial for you to re-test). -e, as far as I understand is to install in development mode, retaining the cloned package dir somewhere under python/src, yes.

**Pull request merged**